### PR TITLE
Clarify Flow Segment in README

### PR DESCRIPTION
**Pull request by GeorginaShippey**
# Details
Reduce use of the term 'segment' in favor of Flow Segment.

# Pivotal Story (if relevant)
Story URL: https://www.pivotaltracker.com/n/projects/2147543/stories/186876593

# Related PRs
_Where appropriate. Indicate order to be merged._

# Submitter PR Checks
_(tick as appropriate)_

- [ ] PR completes task/fixes bug
- [ ] API version has been incremented if necessary
- [ ] ADR status has been updated, and ADR implementation has been recorded
- [ ] Documentation updated (README, etc.)
- [ ] PR added to Pivotal story (if relevant)
- [ ] Follow-up stories added to Pivotal

# Reviewer PR Checks
_(tick as appropriate)_

- [ ] PR completes task/fixes bug
- [ ] Design makes sense, and fits with our current code base
- [ ] Code is easy to follow
- [ ] PR size is sensible
- [ ] Commit history is sensible and tidy

# Info on PRs
The checks above are guidelines. They don't all have to be ticked, but they should all have been considered.

### CODE REVIEW

**Comment by GeorginaShippey** [comment]
I changed some usages of FlowSegment to Flow Segment - any thoughts on that? I think we are generally interested in Flow Segments as a concept rather than an implementation.

Also use of 'media data' or 'grain media data' to talk about the content being stored in the media objects. Any opinions on the best wording - or even if its just best to use the word grain.

File: README.md [hunk 1]
```diff
@@ -49,11 +49,15 @@ At any time the unique address can be exchanged for the media data by an API cal
...
 
-Segments may be stored separately from the metadata linking them to a position on the timeline, separating the metadata and data planes.
-For example our implementation uses an object store (_e.g._ AWS S3) for the segments, passing S3 URLs to the client to upload directly and taking advantage of the scalability of cloud object storage.
-Segments are also de-coupled from their point in the timeline by a link between their `<flow_id, timestamp>` tuple and the underlying object ID, so a single segment can appear at multiple points in multiple flows.
+The grain media data contained within Flow Segments may be stored separately from the metadata linking them to a position on the timeline, separating the metadata and data planes.
+For example our implementation uses a database (_e.g._ Amazon DynamoDB) to store Flow Segment metadata and an object store (_e.g._ AWS S3) to store the grain media data for Flow Segments.
+We refer to grain metadata stored in the object store as 'media objects'.
```
Comment (GeorginaShippey) [inline README.md:56]: Fix: grain metadata -> grain media data

File: README.md [hunk 2]
```diff
@@ -63,18 +67,18 @@ The Flow model is aligned with the principles and schemas of [AMWA NMOS IS-04](h
...
+2. Client makes a request to [`GET flows/<flow_id>/segments?timerange=<timerange>`](https://bbc.github.io/tams/#/operations/GET_flows-flowId-segments) and receives a list of Flow Segments, including their timeranges and download URLs
+3. Client downloads each URL, concatenates the Flow Segments together and unwraps the grains within
 4. The first and last Flow Segment may contain more grains than requested, so the client should skip any received not in the requested timerange
 
 The process of writing to the store is:
 
 1. Client creates a Flow if necessary by making a request to [`PUT flows/<flow_id>`](https://bbc.github.io/tams/#/operations/PUT_flows-flowId)
 2. Client makes a request to [`POST flows/<flow_id>/storage`](https://bbc.github.io/tams/#/operations/POST_flows-flowId-storage) with the timerange to be written
-3. Store responds with a list of segment timeranges and URLs to PUT segments to, along with an optional `pre` URL to call before writing
+3. Store responds with a list of Flow Segment timeranges and URLs to PUT media data into, along with an optional `pre` URL to call before writing
```
Comment (AndrewGibb) [inline README.md:78]: This is OK for now, but will need revisiting after Sam has made his diagrams about Segments, and we've properly shaken down the questions around fixed-size segments. I'll pop in another ticket for that.

**Review by AndrewGibb — CHANGES_REQUESTED** [review]
A few suggested changes. Let me know what you think.

**Comment by samdbmg** [comment]
Changes LGTM, barring Andy's notes

**Review by AndrewGibb — APPROVED** [review]
LGTM

**Pull request merged**